functions/http/send-http-request/src/main/java/functions/SendHttpRequest.java

```diff
@@ -41,17 +41,15 @@ public void service(HttpRequest request, HttpResponse response)
       throws IOException, InterruptedException {
     // Create a GET sendHttpRequest to "http://example.com"
     String url = "http://example.com";
-    java.net.http.HttpRequest getRequest =
-        java.net.http.HttpRequest.newBuilder().uri(URI.create(url)).GET().build();
+    var getRequest = java.net.http.HttpRequest.newBuilder().uri(URI.create(url)).GET().build();
```

Comment:
I have mixed feelings on `var` - it's simpler, but (to me) feels less idiomatic for Java.

@kurtisvg @lesv WDYT?

Reply:
It hasn't been in the language all that long, but I think everyone should use it in appropriate places, of which this is definitely one since it allows us to replace three fully-qualified class names with just one.

Reply:
It should be fine to use in Java 11 apps.

Reply:
It's J11, it's often clearer and less error prone.